docs/en/rules/Azure.APIM.PolicyBase.md

```diff
@@ -0,0 +1,83 @@
+---
+severity: Important
+pillar: Security
+category: Security design principles
+resource: API Management
+online version: https://azure.github.io/PSRule.Rules.Azure/en/rules/Azure.APIM.PolicyBase/
+---
+
+# Base element
+
+## SYNOPSIS
+
+Base element for any policy element in a section should be configured.
+
+## DESCRIPTION
+
+Determine the policy evaluation order by placement of the base element in each section in the policy definition at each scope. The base element inherits the policies configured in that section at the next broader (parent) scope. Otherwise inherited security or other controls may not apply.
+
+The base element can be placed before or after any policy element in a section, depending on the wanted evaluation order.
```

Comment:
```suggestion
The _base_ element inherits the policies configured in that section at the next broader (parent) scope.
Otherwise inherited security or other controls may not apply.
The _base_ element can be placed before or after any policy element in a section, depending on the wanted evaluation order.
However, if security controls are defined in inherited scopes it may decrease the effectiveness of these controls.
For most cases, unless otherwise specified in the policy reference (such as `cors`) the _base_ element should be specified as the first element in each section.

A specific exception is at the _Global_ scope.
The _Global_ scope does not need the _base_ element because this is the peak scope from which all others inherit.
```